sdk/README.md

```diff
@@ -126,6 +126,12 @@ Waiting for logs to be available...
 ```
 
 ## Troubleshooting
-- Please be aware that defined Affinity, Node Selector, and Tolerations are applied to all the tasks in the same pipeline because there's only one podTemplate allowed in each pipeline.
 
+- Here are [some known limitations](kfp-tekton-limitation.md) on the kfp-tekton.
+
+<<<<<<< HEAD
 - When you encounter permission issues related to ServiceAccount, refer to [Servince Account and RBAC doc](sa-and-rbac.md)
+=======
+- When you encounter permission issues related to ServiceAccount, refer to [sa-and-rbac](sa-and-rbac.md)
+
+>>>>>>> initial draft
```

Comment:
can you fix this conflict?

Reply:
thanks, sorry, somehow my editor didn't catch this. done